app/models/plan.rb

```diff
@@ -314,8 +349,7 @@ def add_guidance_to_array(guidance_array, guidance_group, theme, guidance)
   # @return [Boolean] true if user can edit the plan
 	def editable_by?(user_id)
     user_id = user_id.id if user_id.is_a?(User)
-    role = roles.where(user_id: user_id).first
-    return role.present? && role.editor?
+    has_role(user_id, :editor)
```

Comment:
Refactored these because the feedback process assigns the 'reviewer' role to all org admins but they may have also had another role on the plan. The `first` in the old code was preventing these functions from working accurately in that situation

Reply:
well spotted